lib/content/package-json.hbs

```diff
@@ -3,10 +3,20 @@
   "files": {{{ json distPaths }}},
   "type": {{#if esm}}"module"{{else}}{{{ del }}}{{/if}},
   "scripts": {
-    "lint": "{{#if eslint}}eslint \"**/*.{js,cjs,ts,mjs,jsx,tsx}\"{{else}}echo linting disabled{{/if}}",
+    {{#if eslint}}
+    "eslint": "eslint \"**/*.{{{ extGlob lintExtensions }}}\"",
+    "lint": "{{ localNpmPath }} run eslint {{~#if prettier}} && {{ localNpmPath }} run prettier -- --check{{/if}}",
+    "lintfix": "{{ localNpmPath }} run eslint -- --fix {{~#if prettier}} && {{ localNpmPath }} run prettier -- --write{{/if}}",
```

Comment:
Once we've adopted prettier in most repos, we may to separate linting and styling.

We could add dedicated `style` and `stylefix` scripts, and have `posttest` run `{{ localNpmPath }} run lint && {{ localNpmPath }} run style`